version/version.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2014-2022 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
```

Comment:
wanted to put this in the `govmomi` package, but this causes cyclic import.

Reply:
Could also go in `internal/` to not make it part of the API - though I can see users wanting to use this information too.

Reply:
We have so many types/methods/consts/packages already, I like the internal idea for now. We can always move it out later, but not the other way around :)

Reply:
Had to move it into `internal/version` due to cyclic import again.